src/test/java/org/cbioportal/cdd/ClinicalDataDictionaryTestConfig.java

```diff
@@ -49,21 +49,21 @@ public void resetNotWorkingClinicalAttributesRepository(ClinicalAttributeMetadat
         Mockito.when(clinicalAttributesRepository.getClinicalAttributeMetadataOverrides()).thenThrow(new RuntimeException("faking a problem getting the clinical attribute data"));
     }
 
-    private List<ClinicalAttributeMetadata> makeMockAttributeList() {
-        List<ClinicalAttributeMetadata> attributeList = new ArrayList<>();
+    private ArrayList<ClinicalAttributeMetadata> makeMockAttributeList() {
+        ArrayList<ClinicalAttributeMetadata> attributeList = new ArrayList<>();
         attributeList.add(new ClinicalAttributeMetadata("AGE", "Diagnosis Age", "Age at which a condition or disease was first diagnosed.", "NUMBER", "PATIENT", "1"));
         attributeList.add(new ClinicalAttributeMetadata("BONE_MARROW_SAMPLE_HISTOLOGY", "Bone Marrow Sample Histology", "Bone Marrow Sample Histology", "STRING", "SAMPLE", "1"));
         attributeList.add(new ClinicalAttributeMetadata("CLIN_M_STAGE", "Neoplasm American Joint Committee on Cancer Clinical Distant Metastasis M Stage", "Extent of the distant metastasis for the cancer based on evidence obtained from clinical assessment parameters determined prior to treatment.", "STRING", "PATIENT", "1"));
         attributeList.add(new ClinicalAttributeMetadata("DISEASE_STAGE", "Disease Stage", "Disease Stage", "STRING", "SAMPLE", "1"));
         attributeList.add(new ClinicalAttributeMetadata("LAST_STATUS", "Last Status", "Last Status.", "STRING", "PATIENT", "1"));
-        return Collections.unmodifiableList(attributeList);
```

Comment:
I worry a little that this may affect the unit tests in some way, but probably not (unless the test code has side effects which would modify the attributes)